onTruckPlugin/src/main/java/com/github/ontruck/filters/GenericFilter.java

```diff
@@ -0,0 +1,36 @@
+package com.github.ontruck.filters;
+
+import com.github.ontruck.IDriver;
+import com.github.ontruck.MopedState;
+
+public class GenericFilter extends StateFilter {
```

Comment:
It's quite unclear what this class does. Some java-doc explaining it would help.